gslib/commands/acl.py

```diff
@@ -340,6 +343,41 @@ class AclCommand(Command):
       },
   )
 
+  def get_gcloud_storage_args(self):
+    _CannedGcsAcl = ['private', 'public-read', 'project-private',
```

Comment:
Is there already a constant for this somewhere? Why redeclare the constant in each run of this function?